inst/templates/ad_adlb.R

```diff
@@ -259,6 +275,46 @@ adlb <- adlb %>%
     check_type = "error"
   )
 
+# Get WBC
+# Uncomment when derive_param_wbc_abs() is merged (PR #1035)
+# adlb <- adlb %>%
+#   derive_param_wbc_abs(#dataset = input,
+#                        by_vars = vars(USUBJID, VISIT),
+#                        set_values_to = vars(PARAMCD = "LYMPHSI",
+#                                           DTYPE = "CALCULATION"),
+#                        filter_diff = LBSTRESU == "fraction of 1",
+#                        wbc_code = "WBC",
+#                        diff_code = "LYMLE")
+
+# Get extreme values
+# Uncomment when derive_extreme_records() is merged (PR #1044)
+# adlb <- adlb %>%
+#   derive_extreme_records(
+#   #input,
+#   order = vars(AVISITN, LBSEQ),
+#   by_vars = vars(USUBJID),
+#   mode = "last",
+#   set_values_to = vars(DTYPE = "MINIMUM")
```

Comment:
Would we need to set AVISIT and AVISITN also when we call `derive_extreme_records` ?

Reply:
@konstantinakoukR the `filter` needs updated to remove the previous records that `derive_extreme_records` created, ie for 2nd call of `derive_extreme_records` we don't want to look at DTYPE='MINIMUM' records.
Either update the `filter` for 2nd and 3rd calls of `derive_extreme_records` with `is.na(DTYPE)` or update all 3 calls for `filter` and add AVISITN < 9997 (would add a comment why this filter is there). Could also just pipe to each call of `derive_extreme_records`, instead of `adlb <- adlb %>%` for each of the 3 calls, as they can be grouped together. Perhaps a comment about getting the first occurence of the MINIMUM and MAXIMUM (it may not be obvious to users?)